ui-kit/src/main/kotlin/org/p2p/uikit/atoms/UiKitCounterView.kt

```diff
@@ -0,0 +1,80 @@
+package org.p2p.uikit.atoms
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.Gravity
+import androidx.appcompat.widget.AppCompatTextView
+import androidx.core.content.ContextCompat
+import org.p2p.uikit.R
+import org.p2p.uikit.utils.toPx
+
+private const val VISIBLE_COUNT_MAX_VALUE = 99
+private const val MIN_SIZE_DP = 18
+private const val VERTICAL_PADDING_DP = 0
+private const val HORIZONTAL_PADDING_DP = 4
+private const val PLUS_SIGN = '+'
+
+class UiKitCounterView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : AppCompatTextView(context, attrs, defStyleAttr) {
+
+    var count = 0
+        set(value) {
+            text = validateCount(value)
+            field = value
+        }
+
+    var isRead: Boolean = false
+        set(value) {
+            setBackground(value)
```

Comment:
What is `isRead`? 
And why we don't set the color just from xml params?

Reply:
I wanted to encapsulate this two colours in view, because counter can be only one of them. But I'm ok with replacing it with usual background if you want so

Reply:
Okay with colors. 
What is `isRead`? 
What does it mean?

Reply:
maybe `isBadgeVisible`?